src/Microsoft.Azure.WebJobs.Extensions.EventHubs/EventHubConfiguration.cs

```diff
@@ -22,7 +23,9 @@ public class EventHubConfiguration
         // The same path can have multiple connection strings with different permissions (sending and receiving), 
         // so we track senders and receivers separately and infer which one to use based on the EventHub (sender) vs. EventHubTrigger (receiver) attribute. 
         // Connection strings may also encapsulate different endpoints. 
-        private readonly Dictionary<string, EventHubClient> _senders = new Dictionary<string, EventHubClient>(StringComparer.OrdinalIgnoreCase);
+
+        // The client cache must be thread safe because clients are accessed/added on the function
```

Comment:
I don't understand "accessed/added on the function"

Reply:
Uhm, this is your comment that I copied and pasted.... look at https://github.com/Azure/azure-webjobs-sdk/pull/1707